packages/datadog_flutter_plugin/lib/src/rum/ddrum.dart

```diff
@@ -87,9 +87,12 @@ class DatadogRum {
     return DdRumPlatform.instance;
   }
 
-  final sampleRandom = Random();
+  /// The sampling rate for tracing resources.
+  ///
+  /// See [RumConfiguration.tracingSamplingRate]
```

Comment:
I cannot find `RumConfiguration` file in the branch https://github.com/DataDog/dd-sdk-flutter/tree/feature/v2, is it a correct reference? Also it is strange to see notion of `tracing` for RUM here. Should it be in the HTTP plugin support instead only? But I guess it is coming from v1 anyway.

Reply:
Good catch. The configuration was indeed renamed and I missed the doc reference.

Yes, it does only refer to http tracing at the moment, and coming from v1. However, we have several packages that do HTTP tracing and I think it might make sense for them to share the configuration?